tests.py

```diff
@@ -368,9 +368,6 @@ def test_ec2_hosts_inventory(self):
         self.assertFalse(self.output_contains(r.output, '.*ansible-playbook.*-i /etc/ansible/hosts/ec2.py'))
         self.assertEquals(r.exit_code, 0)
         
-        r = self.drun(cmd="cat /etc/ansible/hosts/ec2.ini")
```

Comment:
Not sure why we removed this line as it is here to test the default value is set correctly `default vpc_destination_variable should be private_ip_address`
(maybe this test is overkilled ?)

Reply:
ok, so it means `ec2.ini` is mandatory. I thought it was used only when `AWS_INVENTORY` was true.

Reply:
I think the test since we have auto inventory should be fixed

Cause before your change the ec2.ini was always present so using the default value.
As indeed it does not make sense to always have it, we should update the test.

As you can see this test rely on the default value of https://github.com/cycloidio/docker-cycloid-toolkit/blob/develop/tests.py#L378 `EC2_VPC_DESTINATION_VARIABLE`

We tested once without putting this env var, then a second time with this var set.
So yes maybe you are right just remove this test, it's overkill to add another test section to just validate the default value.

Meanwhile I think you can adapt the aws test like the azure one https://github.com/cycloidio/docker-cycloid-toolkit/blob/develop/tests.py#L405-L409

cause now by default we want to make sure ec2 inventory is not specified when we don't have any access key provided.

From a global point of view, if you don't specify any dynamic inventory stuff related, no inventory should be set on ansible commandline

Reply:
Yep, this is what I wrote in my commit message: 

```
IMHO, this assertion depends on EC2_VPC_DESTINATION_VARIABLE, so it's inside an AWS context.
```

Remove the test is surely a bit excessive so I'm just going to update it.